varats/paper/case_study.py

```diff
@@ -62,17 +62,15 @@ def __repr__(self) -> str:
                     commit_id=self.commit_id)
 
 
-class CSStage(yaml.YAMLObject):
+class CSStage():
     """
     A stage in a case-study, i.e., a collection of revisions. Stages are used
     to separate revisions into groups.
     """
-
-    yaml_loader = yaml.SafeLoader
-    yaml_tag = u'!CSStage'
-
-    def __init__(self, name: tp.Optional[str] = None) -> None:
-        self.__revisions: tp.List[HashIDTuple] = []
+    def __init__(self,
+                 name: tp.Optional[str] = None,
+                 revisions: tp.List[HashIDTuple] = []) -> None:
+        self.__revisions: tp.List[HashIDTuple] = revisions
```

Comment:
```suggestion
        self.__revisions: tp.List[
            HashIDTuple] = revisions if revisions is not None else []
```
adapted change here, github currently does not allow multi line suggestions :)

Reply:
Resolved in 4cd506d